src/hotspot/share/gc/shared/referenceProcessor.cpp

```diff
@@ -1126,13 +1135,7 @@ void ReferenceProcessor::preclean_discovered_references(BoolObjectClosure* is_al
 }
 
 // Walk the given discovered ref list, and remove all reference objects
-// whose referents are still alive, whose referents are NULL or which
-// are not active (have a non-NULL next field). NOTE: When we are
```

Comment:
I think the part of the comment beginning with "NOTE" is still relevant and should not be deleted.  Apologies for not pointing this out earlier.